data/src/test/java/camp/nextstep/edu/github/data/GithubRepositoryImplTest.kt

```diff
@@ -0,0 +1,54 @@
+package camp.nextstep.edu.github.data
+
+import camp.nextstep.edu.github.domain.RepositoryVO
+import com.google.common.truth.Truth.assertThat
+import com.google.gson.GsonBuilder
+import kotlinx.coroutines.runBlocking
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import java.io.FileReader
+
+internal class GithubRepositoryImplTest {
+
+    private lateinit var server: MockWebServer
+    private lateinit var repo: GithubRepositoryImpl
```

Comment:
```suggestion
    private lateinit var repo: GithubRepository
```
인터페이스 타입을 사용하게 되면 구현체의 변경에 영향을 받지 않으며, 동일한 인터페이스로 구현한 구현체도 활용할 수 있어요.
가령, 테스트를 위한 Repository를 활용할 수 있습니다.
타입을 구현체가 아닌 인터페이스 타입으로 변경해보면 어떨까요?